src/components/grid-aware/StatsBlock/StatsBlock.stories.jsx

```diff
@@ -3,33 +3,44 @@ import StatsBlock from "./StatsBlock";
 
 export default {
   title: "Grid-Aware/StatsBlock",
+  subtitle: "Optional Subtitle",
   component: StatsBlock,
 };
 
-const Template = ({ title, statCards }) => (
-  <StatsBlock title={title} statCards={statCards} />
+const Template = ({ title, subtitle, statCards }) => (
+  <StatsBlock
+    title={title}
+    subtitle={subtitle}
+    statCards={statCards}
+    titleColor="blue"
```

Comment:
Ah, this needs to be a prop on the `Template`, like the other props of StatsBlock. Otherwise, when you're in Storybook, attempting to change the title color in the controls section at the bottom of the page won't do anything.

<img width="861" alt="Screen Shot 2022-02-01 at 7 37 23 PM" src="https://user-images.githubusercontent.com/1002748/152089555-851bf7cd-f72f-49bb-83bf-4d8432879a83.png">